scalar.go

```diff
@@ -74,3 +75,29 @@ func IsID(value string) bool {
 	}
 	return strings.HasPrefix(string(decoded), "gid://")
 }
+
+// NullableValue can be used to unset a value using an OpsLevel input struct type, should always be instantiated using a constructor.
+type NullableValue[T any] struct {
+	Value   T
+	SetNull bool
+}
+
+func (nullableValue NullableValue[T]) MarshalJSON() ([]byte, error) {
+	if nullableValue.SetNull {
+		return []byte("null"), nil
+	}
+	return json.Marshal(nullableValue.Value)
+}
+
+func NewNullValue[T any]() *NullableValue[T] {
+	return &NullableValue[T]{
+		SetNull: true,
+	}
+}
+
+func NewNullableValue[T any](value T) *NullableValue[T] {
+	return &NullableValue[T]{
+		Value:   value,
+		SetNull: false,
+	}
+}
```

Comment:
You could potentially make this a global var is you rearrange the constructors like `sql` does

```suggestion
var Null = NewNullable(nil, true)

func NewNullable[T any](value T, isNull bool) *NullableValue[T] {
	return &NullableValue[T]{
	    Value: value
		SetNull: isNull,
	}
}

func NewNullableFrom[T any](value T) *NullableValue[T] {
	return &NullableValue[T]{
		Value:   value,
		SetNull: false,
	}
}
```

Then if you ever need to use just `Null` you can

```
input := opslevel.ServiceUpdateInput2{
    Description: opslevel.Null,
    Framework: opslevel.NewNullableFrom[string]("Cobra")
}
```

Reply:
Renamed and added comment for the constructors